discovery-provider/plugins/pedalboard/apps/relay/src/config.ts

```diff
@@ -19,8 +19,7 @@ export type Config = {
 
 // reads .env file based on environment
 const readDotEnv = () => {
-  // needs to be UPPERCASE because audius-docker-compose
-  const environment = process.env.ENVIRONMENT || "dev";
+  const environment = process.env.audius_discprov_env || "dev";
```

Comment:
is this loaded from the `prod.env` file? or does audius-docker-compose export these somewhere?

Reply:
i believe from respective prod/stage.env files in audius-docker-compose https://github.com/AudiusProject/audius-docker-compose/blob/dev/discovery-provider/prod.env#L16